lib/reporters/cli/index.js

```diff
@@ -134,7 +134,12 @@ PostmanCLIReporter = function (emitter, reporterOptions, options) {
     emitter.on('beforeRequest', function (err, o) {
         if (err) { return; }
```

Comment:
This can be cleanly written as:
```js
        if (err || !o.request) { return; }

        print('  %s %s ', colors.gray(o.request.method), colors.gray(o.request.url));

        !options.disableUnicode && print().wait(colors.gray);

```